server/src/services/notification.service.ts

```diff
@@ -100,8 +104,30 @@ export class NotificationService extends BaseService {
   }
 
   @OnEvent({ name: 'album.update' })
-  async onAlbumUpdate({ id, updatedBy }: ArgOf<'album.update'>) {
-    await this.jobRepository.queue({ name: JobName.NOTIFY_ALBUM_UPDATE, data: { id, senderId: updatedBy } });
+  async onAlbumUpdate({ id, recipientIds }: ArgOf<'album.update'>) {
+    // if deliverTo is empty, album likely only has one user part of it, don't queue notification if so
+    if (!recipientIds) {
```

Comment:
Arrays are truthy, even empty ones.

Reply:
oh now IDE is saying `recipientIds` being `undefined` is impossible, I'll just keep the length check only, am too used to Java world trusting no one 🙃